pkg/jsonfieldstest/jsonfieldstest.go

```diff
@@ -0,0 +1,29 @@
+package jsonfieldstest
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/cli/cli/v2/pkg/cmdutil"
+	"github.com/spf13/cobra"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// May God have mercy on my soul
```

Comment:
lolololololol

Reply:
Removed this and replaced it with useful comments/